app/actions/get_user_from_omniauth_action.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class GetUserFromOmniauthAction < ApplicationAction
```

Comment:
Please note that the term "get" typically does not imply any additional actions beyond the intended task, such as saving data to a database. It might be worth considering renaming this function to a more descriptive term, such as "OmniauthAuthenticateAction," to better reflect its purpose. "Authenticate" is a more abstract and comprehensive term, which includes subactions such as implicit signup.